File: 프로그래머스/unrated/120904. 숫자 찾기/숫자 찾기.py
def solution(num, k):
    str_num = str(num)  # 정수를 문자열로 변환
    for i, digit in enumerate(str_num):
        if int(digit) == k:
            return i + 1  # 처음으로 일치하는 숫자의 위치를 반환
    return -1  # 일치하는 숫자가 없으면 -1을 반환

File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        int answer = 0;
        int len = (int) Math.log10(num)+1;
         int array[] = new int[len];
        for(int i=0;i<len;i++){
            array[len-i-1]=num%10;
            num /=10;
        }
        for(int j=0;j<len;j++){
            if(array[j]==k){
                answer= j+1;
                break;
            }else{
                answer=-1;
            }
        }
        return answer;
    }
}